package/network/utils/iptables/Makefile

```diff
@@ -598,6 +598,7 @@ endef
 define Package/iptables/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/xtables-legacy-multi $(1)/usr/sbin/
+	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/iptables-legacy{,-restore,-save} $(1)/usr/sbin/
```

Comment:
Please add a commit message describing what you're doing. I'm guessing building `iptables` generates symlinks already, right?

Reply:
Improved a bit the commit message
```
 iptables: add ip{,6}tables-legacy{,-restore,-save} symlinks

Now that we can have both legacy and nft iptables variants
installed at the same time, install the legacy symlinks
```

Reply:
yup iptables build generate the symlinks